<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>564</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="whatsThis">
   <string>主对话框
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="ImageButton" name="btnSysStting">
        <property name="text">
         <string>系统设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ImageButton" name="pushButton_2">
        <property name="text">
         <string>工作测试</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ImageButton" name="pushButton_3">
        <property name="text">
         <string>数据查询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ImageButton" name="pushButton_4">
        <property name="text">
         <string>系统调试</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ImageButton" name="pushButton_5">
        <property name="text">
         <string>炉温校正</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ImageButton" name="pushButton_6">
        <property name="text">
         <string>帮助</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ImageButton" name="pushButton_7">
        <property name="text">
         <string>退出系统</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ImageButton" name="pushButton_8">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>26</number>
        </property>
        <item>
         <widget class="ImageButton" name="pushButton_12">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ImageButton" name="pushButton_10">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ImageButton" name="pushButton_9">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,14">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="TabMenuControl" name="tabMenu">
        <property name="whatsThis">
         <string>管理二级菜单的tab控件
</string>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::South</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="tabBarAutoHide">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="tabMenuPage1">
         <attribute name="title">
          <string/>
         </attribute>
        </widget>
        <widget class="QWidget" name="tabMenuPage2">
         <attribute name="title">
          <string/>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalTab" stretch="7,3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTabWidget" name="tabGrid">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <property name="usesScrollButtons">
           <bool>true</bool>
          </property>
          <property name="tabBarAutoHide">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabCurve">
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageButton</class>
   <extends>QPushButton</extends>
   <header>imagebutton.h</header>
  </customwidget>
  <customwidget>
   <class>TabMenuControl</class>
   <extends>QTabWidget</extends>
   <header>tabmenucontrol.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>875</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>876</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_10</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showFullScreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>837</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>837</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_12</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>809</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>809</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
